source/gui/settingsDialogs.py

```diff
@@ -1724,214 +1724,166 @@ def makeSettings(self, settingsSizer):
+			# Translators: Label for an option in the font list in the document formatting settings panel.
+			"reportFontSize": _("Font size"),
+			# Translators: Label for an option in the font list in the document formatting settings panel.
+			"reportFontAttributes": _("Font attributes"),
+			# Translators: Label for an option in the font list in the document formatting settings panel.
+			"reportEmphasis": _("Emphasis"),
+			# Translators: Label for an option in the font list in the document formatting settings panel.
+			"reportStyle": _("Style"),
+			# Translators: Label for an option in the font list in the document formatting settings panel.
+			"reportColor": _("Colors"),
+		}
+		self.fontList = sHelper.addLabeledControl(
+			fontListText,
+			nvdaControls.CustomCheckListBox,
+			choices=list(self.fontListMap.values())
+		)
+		self.fontList.CheckedItems = [
+			i for i, setting in enumerate(self.fontListMap)
+			if config.conf["documentFormatting"][setting]
+		]
+		self.fontList.Select(0)
 
-		# Translators: This is the label for a group of document formatting options in the 
+		# Translators: This is the label for a list with document formatting options in the
 		# document formatting settings panel
-		documentInfoGroupText = _("Document information")
-		docInfoGroup = guiHelper.BoxSizerHelper(self, sizer=wx.StaticBoxSizer(wx.StaticBox(self, label=documentInfoGroupText), wx.VERTICAL))
-		sHelper.addItem(docInfoGroup)
-
-		# Translators: This is the label for a checkbox in the
-		# document formatting settings panel.
-		commentsText = _("Co&mments")
-		self.commentsCheckBox=docInfoGroup.addItem(wx.CheckBox(self,label=commentsText))
-		self.commentsCheckBox.SetValue(config.conf["documentFormatting"]["reportComments"])
-
-		# Translators: This is the label for a checkbox in the
-		# document formatting settings panel.
-		revisionsText = _("&Editor revisions")
-		self.revisionsCheckBox=docInfoGroup.addItem(wx.CheckBox(self,label=revisionsText))
-		self.revisionsCheckBox.SetValue(config.conf["documentFormatting"]["reportRevisions"])
-
-		# Translators: This is the label for a checkbox in the
-		# document formatting settings panel.
-		spellingErrorText = _("Spelling e&rrors")
-		self.spellingErrorsCheckBox=docInfoGroup.addItem(wx.CheckBox(self,label=spellingErrorText))
-		self.spellingErrorsCheckBox.SetValue(config.conf["documentFormatting"]["reportSpellingErrors"])
+		docInfoListText = _("&Document information")
+		self.docInfoListMap = {
+			# Translators: Label for an option in the document information list in the
+			# document formatting settings panel.
+			"reportComments": _("Comments"),
+			# Translators: Label for an option in the document information list in the
+			# document formatting settings panel.
+			"reportRevisions": _("Editor revisions"),
+			# Translators: Label for an option in the document information list in the
+			# document formatting settings panel.
+			"reportSpellingErrors": _("SPelling and grammar errors"),

```

Comment:
Fix typo: s/SPelling/spelling

```suggestion
			"reportSpellingErrors": _("Spelling and grammar errors"),
```